os/android/gralloc1bufferhandler.cpp

```diff
@@ -200,7 +214,15 @@ bool Gralloc1BufferHandler::ImportBuffer(HWCNativeHandle handle) const {
   gralloc1_device_t *gralloc1_dvc =
       reinterpret_cast<gralloc1_device_t *>(device_);
   register_(gralloc1_dvc, handle->imported_handle_);
-  return ImportGraphicsBuffer(handle, fd_);
+  if (ImportGraphicsBuffer(handle, fd_)) {
+    if (handle->rbc_enabled_) {
+      handle->meta_data_.numplanes = 2;
+      handle->meta_data_.modifier = handle->modifier_;
+    }
+  } else {
```

Comment:
you can avoid else here, just return true here and line 225 can return false

Reply:
will do.